make-util.js

```diff
@@ -0,0 +1,201 @@
+
+require('shelljs');
+var fs = require('fs');
+var os = require('os');
+var path = require('path');
+var process = require('process');
+var ncp = require('child_process');
+var syncRequest = require('sync-request');
+
+var downloadPath = path.join(__dirname, '_download');
+var testPath = path.join(__dirname, '_test');
+
+var banner = function (message, noBracket) {
+    console.log();
+    if (!noBracket) {
+        console.log('------------------------------------------------------------');
+    }
+    console.log(message);
+    if (!noBracket) {
+        console.log('------------------------------------------------------------');
+    }
+    console.log();
+}
+exports.banner = banner;
+
+var rp = function (relPath) {
+    return path.join(pwd() + '', relPath);
+}
+exports.rp = rp;
+
+var fail = function (message) {
+    console.error('ERROR: ' + message);
+    process.exit(1);
```

Comment:
it would be nice longer-term to bubble out, or to an outer catch handler. the concern is that buffered stdout will be lost. however i believe this fail method is the same as it is today, so not any worse.